services-geocoding/src/main/java/com/mapbox/api/geocoding/v6/models/V6Context.java

```diff
@@ -0,0 +1,113 @@
+package com.mapbox.api.geocoding.v6.models;
+
+import androidx.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * V6 response's piece of data which breaks out the complete geographical hierarchy
+ * for a given address or place. It is both a reliable way to access the named values of each
+ * component part of an address, plus contains feature-specific data such as the Wikidata id and
+ * 3-letter alpha country code.
+ *
+ * @see <a href="https://docs.mapbox.com/api/search/geocoding-v6/#the-context-object">The Context Object</a>
+ */
+@AutoValue
+public abstract class V6Context extends V6JsonObject {
+
+  /**
+   * Address context element.
+   *
+   * @return address context element
+   */
+  @SerializedName("address")
+  @Nullable
+  public abstract V6ContextElement address();
```

Comment:
With `V6ContextElement` users will have direct access only to `id` and `name` whereas [documentation](https://docs.mapbox.com/api/search/geocoding/#the-context-object) say that 

> Address features will also include an address sub-object, which additionally contains street_name and address_number properties

I'd recommend make sure public documentation is up to date and then compare all the v6 API with what's in PR

Reply:
I added this one, which wasn't something new, these fields were always there. Anyway, I found other differences between the context in the PR and in the public documentation, but it's all some fields which are missing. I spoke to Robert and we're okay with having some differences or missing features for now. We can always add that later as long as nothing is wrong now, the top priority is to have a release by end of this week.

Reply:
I added this one.